src/components/authentication/register/RegisterForm.js

```diff
@@ -98,6 +124,28 @@ export default function RegisterForm() {
             helperText={touched.email && errors.email}
           />
 
+          <TextField
+            fullWidth
+            label="Address"
+            {...getFieldProps('address')}
+            error={Boolean(touched.address && errors.address)}
+            helperText={touched.address && errors.address}
+          />
+
+          <TextField
+            fullWidth
+            label="Phone number"
+            {...getFieldProps('phone_number')}
+            error={Boolean(touched.phone_number && errors.phone_number)}
+            helperText={touched.phone_number && errors.phone_number}
+          />
+
+          <RadioGroup row {...getFieldProps('gender')}>
```

Comment:
dd one more radio button and label it "Not  Applicable". This is because an Organization does not have a gender.